src/api/spec/controllers/webui/apidocs_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Webui::ApidocsController, type: :controller do
+  describe "GET index" do
+    context "correct setup" do
+      render_views
+
+      before do
+        get :index
+      end
+
+      it { expect(response.body).to have_title('Open Build Service') }
+    end
+
+    context "broken setup" do
+      let!(:old_location) { CONFIG['apidocs_location'] }
+
+      before do
+        CONFIG['apidocs_location'] = '/your/mom'
+      end
+
+      after do
+        CONFIG['apidocs_location'] = old_location
+      end
+
+      it "errors and redirects" do
+        expect(Rails.logger).to receive(:error).with(
```

Comment:
why not adding this and the `get :index` in the `before`? This way you leave inside the test the things that you want actually to test. I think it is more readable and something it is generally done in the project.